ui/src/components/SecretsTable/index.jsx

```diff
@@ -89,7 +89,7 @@ export default class SecretsTable extends Component {
   }
 
   render() {
-    const { onPageChange, classes, secretsConnection } = this.props;
+    const { onPageChange, classes, secretsConnection, searchTerm } = this.props;
```

Comment:
`searchTerm` should be added to `propTypes` and `defaultProps`. The default value can be null. Same with the other ones.

```jsx
propTypes = {
  searchTerm: string,
};

defaultProps = {
  searchTerm: null,
};
```